src/Components/Onboarding/Hooks/useOnboardingContext.tsx

```diff
@@ -73,6 +75,10 @@ const OnboardingContext = createContext<{
   current: string
   dispatch: React.Dispatch<Action>
   next(): void
+  // Called when the user has finished the onboarding process
+  onComplete(): void
+  // Called on close and other cancel actions
+  // TODO: Should we rename this to onClose?
```

Comment:
Is this called whenever someone closes onboarding, or do we check to see if they've actually filled out all answers?

If the former, I would rename.

Reply:
That's the thing. I would probably defer to @dzucconi when he is back on the thinking here. So far `onDone` is seemingly being used as close, because we need a mechanism to close the modal in a few different places, from a centralized context. I added `onComplete` in order to differentiate (ie, fire `onComplete` inside of a `useEffect` when @TMcMeans 's new thank you screen mounts)

Reply:
Well, it seems like "onComplete" is appropriate here. Complete means that the process of onboarding is finished, closed or I guess "done" in this case, means the user left without completing. 

In this case, then, I actually don't like the use of "onDone" and would change that to "onClose" instead.

Reply:
Digging into this a bit further, and noticed this line: https://github.com/artsy/force/blob/main/src/Components/Onboarding/config.ts#L44-L48

So this seems right: 
- When the workflow is "done", call `onClose` (close the modal)
- Workflow can be "done" any number of ways, including by a user exiting early -- so close the modal
- This is different than `onComplete`, which says: user has successfully completed the onboarding process.

So in sum, will rename 😄

Reply:
We did it! 💃🏽